src/ui/public/field_editor/field_editor.html

```diff
@@ -7,9 +7,10 @@
     </p>
   </div>
   <div ng-if="editor.creating" class="form-group">
-    <label>Name</label>
+    <label for="scriptedFieldName">Name</label>
     <input
       ng-model="editor.field.name"
+      id="scriptedFieldName00------------"
```

Comment:
Btw, I forgot to add that comment yesterday: What has happened to this `id`? 🐱 ⌨️